### DISCUSSION TITLE
Github Student Education Pack Form Bug

### DISCUSSION THREAD

**Discussion by hardianlawi** [community/community#92808] [GitHub Education]
### Hi everyone,

https://github.com/orgs/community/discussions/58970#discussioncomment-8195404

<img width="605" alt="image" src="https://github.com/community/community/assets/22806872/8ee7ad2a-73f3-4d93-ae21-df8bae36d4c9">

There's no option for me to select proof type at all. Is this a bug?

Also, why is there no more explanation about the school and location detected? I'm currently enrolled in an online master programme so it's hard to tell why am I actually being rejected based on the information given since I'm sure the documents I submitted would be approved if the verifiers are actual humans. :')

**Comment by Mohamed-Aziz-Yahia** [comment]
It happend for me too. I still don't get what the problem is, but it seems some kind of location problem, for example when I first tried it was on my PC browser which was super private that it didn't share any lovation, so I got some issues there, but then I switched to my phone and opened location access, it worked. Even though I didn't receive my benefits yet (it's been 3 days), but I took the proof by my phone's front camera because of this bug. I hope they fix it soon cuz it drove me nuts!
